rechunker/api.py

```diff
@@ -70,7 +89,12 @@ def execute(self, **kwargs):
         :func:`rechunker.rechunk`.
         """
         self.compute(**kwargs)
-        return self._target
+        if self._target_original is None:
+            return self._target
+
+        target = zarr.open(self._target_original)
+        target.append(self._target)
```

Comment:
I must admit, I don't understand why we are calling append here. Shouldn't the append operation happen inside the `self.compute` call?

Reply:
My idea was that when we append (`target_append=True`), we want to append new source data to an already existing zarr `target_store`. So we just rechunk the new source data to a temporary target store (which is `'to_append_' + target_store` for now just to make it simple), that is created with the `self.compute` call, and we use zarr's `append` method to append it to the original `target_store`. Do you think this is too expensive, and that we should directly append to the original target store?

Reply:
I think it is best of we try to append directly to the original target store.

Also, we want to have all the computations inside the dask graph. As currently coded (append in execute), this part of the operation will not be run by dask, but rather will run from the client. That will be a huge bottleneck for large arrays.

Reply:
Agreed that we want this to happen inside the compute, so it should be added as a task.

Reply:
> so it should be added as a task.

But not just a single task. We need to parallelize the append operations just like we do all other writes, via `dask.array.to_zarr`.